compiler/test/dotty/tools/TestSources.scala

```diff
@@ -27,12 +27,14 @@ object TestSources {
 
   def runFromTastyBlacklistFile: String = "compiler/test/dotc/run-from-tasty.blacklist"
   def runTestPicklingBlacklistFile: String = "compiler/test/dotc/run-test-pickling.blacklist"
+  def runScalaJSBlacklistFile: String = "compiler/test/dotc/run-scala-js.blacklist"
```

Comment:
Use `// scalajs: --skip` in the test file instead.

Reply:
Done